### PR TITLE
Support what’s new on Apple Store

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: app/models/api_model.php [hunk 1]
```diff
@@ -66,8 +66,9 @@
             // Include the current template
             require TEMPLATES . $project->getTemplate($store, $channel);
 
-            // Google has a 500 characters limit for the What's New section
-            if ($store == 'google' && $set_limit(500, $whatsnew($translations))) {
+            // Only Google has a 500 characters limit for the What's New section
+            if ($store == 'google' && $set_limit(500, $whatsnew($translations))
+                || $store == 'apple') {
```
Comment (pascalchevrel) [inline app/models/api_model.php]: I would prefer a switch. Note that you don't need to split the condition, the 80 character limit is not a PHP rule, in PHP 80 is the advised length but the upper limit is 120. In this case I think it's more readable to have an if statement on a 100 characters line than one on 2 lines.

File: app/views/apple/release/locale_escaped_view.php [hunk 2]
```diff
@@ -5,6 +5,9 @@
 <h3>Description</h3>
 <pre contenteditable="true"><?= strip_tags($description($translations)) ?></pre>
 
+<h3>What’s New in Version 2.0</h3>
```
Comment (pascalchevrel) [inline app/views/apple/release/locale_escaped_view.php]: I think this sentence should be part of the strings to translate

**Comment by pascalchevrel** [comment]
please add functional test for apple/whatsnew/release/ as well as for apple/translation/release/fr/ to check that there is a whatsnew field in the json produced. Thanks

**Comment by TheoChevalier** [comment]
fixed and rebased :beers:

**Pull request merged**